test/utilities.js

```diff
@@ -22,3 +22,9 @@ function child(test, depth) {
   );
   test.strictSame(common.callerFilename(depth), 'utilities.js');
 }
+
+metatests.testSync('Check captureMaxStack', test => {
+  const stack = common.captureMaxStack();
+  test.assert(stack.length > 0);
+  test.type(stack, 'string');
```

Comment:
Can you add some kind of regular expression check here to validate that it is a stack trace, and not some kind of random non-empty string?

Reply:
Done.